cpp-package/include/mxnet-cpp/CPPLINT.cfg

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
we had cpplint in sanity check before so it's probably a good idea to bring it back.